### DISCUSSION TITLE
Tutorial/example for wrapping or migrating a commandline application

### DISCUSSION THREAD

**Discussion by eliotb** [Textualize/textual#2335] [Q&A]
Is there a tutorial or example of an ordinary commandline application being wrapped in a Textual gui?
Preferably without modifying the commandline app code 

I.e. cli options -> textual widgets, stdout, stderr -> same or different TextLog,  user input -> stdin etc. ?

**Comment by davep** [comment]
Funny you should mention that... I don't know of anything out there at the moment but I can confidently say "watch this space". 😏
